Make-a-full-stack-change.md

```diff
@@ -0,0 +1,431 @@
+## Table of Contents
+
+* [Support changing user experience](#support-changing-user-experience)
+  * [Frontend changes part 1](#frontend-changes-part-1)
+  * [Backend changes part 1](#backend-changes-part-1)
+* [Support viewing user experience](#support-viewing-user-experience)
+  * [Backend changes part 2](#backend-changes-part-2)
+  * [Frontend changes part 2](#frontend-changes-part-2)
```

Comment:
Rename headers:

* Support editing
  * Frontend changes to support editing
  * Backend changes to support editing
* Support viewing
  * Backend changes to support viewing
  * Frontend changes to support viewing

Reply:
done